plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergSystemTables.java

```diff
@@ -683,4 +687,31 @@ private BaseTable loadTable(String tableName)
     {
         return IcebergTestUtils.loadTable(tableName, metastore, fileSystemFactory, "hive", "tpch");
     }
+
+    @Test
+    public void testPartitionColumns()
+    {
+        String tableName = "test_partition_columns_" + randomNameSuffix();
+        assertUpdate(String.format("""
+                CREATE TABLE %s WITH (partitioning = ARRAY[
+                    '"r1.f1"',
+                    'bucket(b1, 4)'
+                ]) AS
+                SELECT
```

Comment:
Could you remove unrelated columns from this SELECT statement?

Reply:
removed